src/projections/analysis/IntervalData.java

```diff
@@ -177,8 +179,39 @@ public void loadIntervalData(long intervalSize, int intervalStart,
 	}
     }
 
-    public int[][] sumDetailData() {return  sumDetailData; };
-
+    public int[][] sumDetailData() {
+		int[][] temp = new int[sumDetailData[0].length][sumDetailData[0][0].length];
+		for(int i =0;i<sumDetailData.length;i++){
+			for(int j =0;j<sumDetailData[0].length;j++){
+				for(int k =0;k<sumDetailData[0][0].length;k++){
+					temp[j][k]+=sumDetailData[i][j][k];
+				}	
+			}	
+		}
+		return  temp; 
+	};
+    public int[][] sumDetailDataperproc() {
+		int[][] temp = new int[sumDetailData.length][sumDetailData[0][0].length];
+		for(int i =0;i<sumDetailData.length;i++){
+			for(int j =0;j<sumDetailData[0].length;j++){
+				for(int k =0;k<sumDetailData[0][0].length;k++){
+					temp[i][k]+=sumDetailData[i][j][k];
+				}	
+			}	
+		}
+		return  temp;  
+	};
+    public int[][] sumDetailDataperprocbytime() {
+		int[][] temp = new int[sumDetailData.length][sumDetailData[0].length];
+		for(int i =0;i<sumDetailData.length;i++){
+			for(int j =0;j<sumDetailData[0].length;j++){
+				for(int k =0;k<sumDetailData[0][0].length;k++){
+					temp[i][j]+=sumDetailData[i][j][k];
+				}	
+			}	
+		}
+		return  temp; 
+	};
```

Comment:
These should have more descriptive names indicating what they output, maybe something like `getSumDetailDataIntervalEP`, `getSumDetailDataPEEP`, and `getSumDetailDataPEInterval`.

Similarly, the iteration variables in the loops shouldn't just be `i,j,k`, make them describe what they represent, so things like `pe, interval, ep`.